view/src/main/java/cmm/apps/esmorga/view/eventdetails/EventDetailsView.kt

```diff
@@ -59,7 +59,8 @@ fun EventDetailsScreen(
 ) {
     val uiState: EventDetailsUiState by edvm.uiState.collectAsStateWithLifecycle()
     val context = LocalContext.current
-    val message = stringResource(R.string.snackbar_event_joined)
+    val joinEventSuccessMessage = stringResource(R.string.snackbar_event_joined)
```

Comment:
no need to add a variable here, right? you can call stringResource already where it is used.

Reply:
I need it because stringResource() method needs to be in a @composable function